ui/src/app/shared/models.ts

```diff
@@ -37,6 +37,7 @@ export interface RollbackOperation {
 export interface Operation {
     sync: SyncOperation;
     rollback: RollbackOperation;
+    ignoreWarnings: boolean;
```

Comment:
is this existing on the backend?

Reply:
Nope, just on the UI. If the user never sets it it resolves to `undefined`

Reply:
Just realized that this is a left-over from a previous implementation I had -- I have removed it from the code.